frontend/src/styles/global.css

```diff
@@ -33,8 +33,7 @@ select:-webkit-autofill:focus {
   transition: all 150ms;
 }
 .ant-layout-sider-collapsed .App__title {
-  width: 0;
-  opacity: 0;
+  transform: rotate(-90deg) translate(-60px, -10px);
```

Comment:
Haha that's cute. A suggestion, maybe having something like a circled F, which is a design often used for sidebar panels such as these, could be conceivable too :)

Reply:
Like the new commit?

Reply:
I noticed about your new commit that when the panel is displayed by default, the letters overlap like this;
![image](https://user-images.githubusercontent.com/1617741/112637717-d4772400-8e14-11eb-82ef-917513622e4b.png)

Also, personally Im not a fan of rotating animation, but maybe others have different opinions about that :P

Reply:
Fixed